config/presets/testnet.go

```diff
@@ -102,6 +102,10 @@ func testnet() config.Config {
 			ATXGradeDelay:       30 * time.Minute,
 
 			PprofHTTPServerListener: "localhost:6060",
+
+			AtxVersions: activation.AtxVersions{
+				types.EpochID(2): types.AtxV2,
+			},
```

Comment:
With this being different than the actual value on the current testnet, everybody who wants to join the testnet will need to override in the config.

Reply:
We ship the configs anyway, there is no "testnet" config hardcoded here, it's just defaults that then are overridden PER network anyway. I think it's fine.